erpnext/accounts/doctype/asset_category/test_asset_category.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable */
+// rename this file from _test_[name] to test_[name] to activate
+// and remove above this line
+
+QUnit.test("test: Asset Category", function (assert) {
```

Comment:
This very crazy. Because you are saving add this file but in the same time as I can tell you will not use this doctype in accounts module. Please review the deleted files